lib/DDG/Spice/GetEvents/Nearby.pm

```diff
@@ -1,15 +1,15 @@
-package DDG::Spice::GetEvents;
+package DDG::Spice::GetEvents::Nearby;
```

Comment:
@Uriger yup you can have dynamic caching.

If the query is a "nearby" query, where we don't want to cache the nginx re-write, we would:

```perl
spice is_cached => 1;
spice proxy_cache_valid => "200 12h";
...
if (is_local) {
    return $result, {is_cached => 0};
} else {
        return $result;
}
```